tests/webpack-test/fixtures/node_modules/complexm/step1.js

```diff
@@ -0,0 +1 @@
+module.exports = require("m1/a") + require("m1");
```

Comment:
Why we need add `step1.js`, `step2.js`, `m1/a.js`, `m1/b.js`.... files? 
Where are them using?

Reply:
all files in node_modules/complexm are dead files.

deldeted.